src/Microsoft.Diagnostics.Runtime/src/Common/ModuleInfo.cs

```diff
@@ -65,6 +65,8 @@ public virtual bool IsManaged
                 InitData();
                 return _managed;
             }
+
+            set => _managed = value;
```

Comment:
Can you mark this `internal` if you don't need it to be public?  I plan to refactor this class one day to fix a bunch of stuff and it's easier if there's less external dependencies.  If you actually need it to be `public` that's fine too.

Reply:
Done.